.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Run tests
+
+on: [push]
```

Comment:
This should be also triggered on `pull_request` event so it works on PRs?
And `push` event should probably [exclude](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#excluding-branches-and-tags) `master` branch - so we don't run tests on master after PR is merged?

Reply:
Keeping test when merging on master is good to make sure that nothing got wrong, and to have the ✔︎sign on our last commit which is a good security when someone reaches out saying master is broken ;)

Reply:
It already works on PRs using `push`, from what I experienced on another project

Reply:
@LaRiffle hmm, that's strange. Technically this is a push to a forked repo.

Reply:
Oh yes on a forked repo it might not work :/
But on branches from OpenMined/Pysyft it should (but they are not a majority)